app/views/comments/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1><%= link_to @post.title, @post.link %></h1>
+
+<!--  Add modal for new comment -->
+<a href="#" class="btn btn-lg btn-primary" 
+   data-toggle="modal" 
+   data-target="#newComment">New Comment</a>
+<div id="newComment" class="modal fade" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="gridSystemModalLabel">New Comment</h4>
+      </div>
+      <div class="modal-body">
+        <%= bootstrap_form_for [@post, @comment] do |f| %>
+          <%= f.text_area :content %>
+          <%= f.primary %>
+        <% end %>
+      </div>
+    </div><!-- /.modal-content -->
+  </div><!-- /.modal-dialog -->
+</div><!-- /.modal -->
+
+
+<!-- Show all comments for this post -->
```

Comment:
To get the partial to show for each comment, make sure you nest it in the .each iterator for comments. Also, you must include the post that you are specifying in your locals object.
```
<!-- Show all comments for this post -->
<% @post.comment.each do |comment| %>
<div class="well">
  <p><%= comment.content %></p>
  <strong>By <%= comment.user.name %></strong>
 <%= render partial: 'comments/comment_votes', locals: {comment: comment, post: @post} %>
</div>

<% end %>
```